addons/base/views.py

```diff
@@ -851,10 +851,9 @@ def addon_view_or_download_file(auth, path, provider, **kwargs):
         raise HTTPError(http_status.HTTP_400_BAD_REQUEST)
 
     if hasattr(target, 'get_addon'):
-
         node_addon = target.get_addon(provider)
         if flag_is_active(request, features.ENABLE_GV):
-            if not isinstance(node_addon, EphemeralNodeSettings):
+            if not isinstance(node_addon, EphemeralNodeSettings) and provider != 'osfstorage':
```

Comment:
@brianjgeiger As you have pointed out [here](https://github.com/CenterForOpenScience/osf.io/pull/10750/files#r1757014728), we do have to check for osfstorage as well.